### PR TITLE
WIP Swag Store Case Study Post

### PR DESCRIPTION
**Pull request by marisamorby**

### CODE REVIEW

**Comment by shannonbux** [comment]
PS I meant to leave that as a comment, not approval, b/c I don't manage approving the blog!

File: docs/blog/2018-11-27-swag-store/index.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ So this led us to the idea of creating an automated system that would identify,
 
 We settled on an “automation first” solution for a few reasons
 
-### **Time was limited**
+### Time was limited
 
 We needed to limit manual touchpoints as much as possible in the interest of making sure that people were able to focus on critical Gatsby needs.
```
[the remark this answers is not in the document]
Comment (marisamorby) [reply docs/blog/2018-11-27-swag-store/index.md]: Made this change.

File: docs/blog/2018-11-27-swag-store/index.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+
+Gatsby is a React-based site generator that creates some of the fastest websites you’ve ever seen. Go ahead and blink—I dare you! It’s a framework that not only helps people create super fast websites, but it’s also a great equalizer of the web. Think about it: if you’re in an area that has 3G internet, many sites can take about 15 seconds to load, if they even load at all. Thanks to Gatsby’s performance-first architecture, the same site can be on-screen in less than 3 seconds. So Gatsby can really bring the information to everyone.
+
+Gatsby is also open source and currently has 1000+ contributors. For all the time, energy, and help they’ve given, we wanted to find a way to thank them properly.
+
+But how could we do this? Our contributors live all around the world—on 6 continents!
+
+After a lot of discussion, we settled on giving people free swag! It was a great way to show our appreciation and give away something useful as well.
+
+With that in mind, we needed to find a way to get swag into the hands of contributors quickly and reliably, with no human bottlenecks. The rest of this article is a breakdown of how we made that happen.
```
[the remark this answers is not in the document]
Comment (marisamorby) [reply docs/blog/2018-11-27-swag-store/index.md]: Lol! I can't think of a better way to get this point across, unfortunately.

**Comment by marisamorby** [comment]
@shannonbux: I agree that the post is really long. I pulled this over from Medium where I originally wrote it, but future posts will be more focused on one specific thing rather than a whole project. :)

**Pull request merged**